### DISCUSSION TITLE
Toogle to database

### DISCUSSION THREAD

**Discussion by VurgunGurbanov** [filamentphp/filament#5307] [Help] [answered]
Hi everyone. 
How I can use toogle for insert data( 0 1) to database?
Toggle::make('is_active')
I write code as above but it doesn't work :( . It works as button but not change data on database.

**Comment by ArtMin96** [comment]
Add this to the user model

```php
protected $casts = [
    'is_active' => 'boolean',
];
```

**Reply by VurgunGurbanov** [reply]
I was do it, but it was same. Not working

**Reply by ArtMin96** [reply]
What value do you get when saving?

**Reply by VurgunGurbanov** [reply]
If I don't make a mistake it returns null. Because it is not change the value on the database.

**Reply by ArtMin96** [reply]
Is the table column int or boolean?

**Reply by VurgunGurbanov** [reply]
Boolean

**Comment by RX-77IT** [comment, marked as answer]
If you are using $fillable in your Model, is is_active inside $fillable?
```php
protected $fillable = [
  // other variable names
  'is_active',
];
```

**Reply by VurgunGurbanov** [reply]
Thank you very much, bro! I was search everywhere on network for solution about 2 days. I have forgotten to try it.
